.github/workflows/scheduled-cycle-secrets.yml

```diff
@@ -0,0 +1,128 @@
+name: "[Scheduled] Cycle Secrets"
+
+on:
+  schedule:
+    # 5AM from Sunday to Friday
+    - cron: "0 5 * * 0-5"
```

Comment:
hey @jamesrwarren - any reason skip saturday?

Reply:
Yeah this is not quite right actually. Going to update to Sun-Thur. It's to protect us from some issue breaking it over the weekend where we have no support basically. I can't see what exactly might break but as we're doing TF applies as part of this rotation there's always something that might break (one thought is where an updated version of something that wasn't pinned properly gets applied). We're somewhat protected as it does the environments one after the other rather than in parallel but still a possibility of differences.